core-bundle/src/Exception/BadRequestException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpFoundation\Exception\BadRequestException as BaseBadRequestExceptionAlias;
+
+class BadRequestException extends BaseBadRequestExceptionAlias
```

Comment:
```suggestion
use Symfony\Component\HttpFoundation\Exception\BadRequestException as BaseBadRequestException;

class BadRequestException extends BaseBadRequestException
```

I don’t think we use an `Alias` suffix anywhere else for such `use` statements.
Just a nitpick though. ☺️